content/Software Support/Arduino App Lab/Code-repositories-for-Arduino-App-Lab-components.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Code repositories for Arduino App Lab components"
+---
+
+The Arduino App Lab is composed of several software components, each with its own code repository on GitHub. There, you can browse and download the source code for any component. You can also track ongoing issues and report bugs or propose enhancements.
+
+The table below lists each component, its purpose, and a link to its GitHub repository.
+
+| Component                  | Purpose                                               | Repository                                                                      |
+|----------------------------|-------------------------------------------------------|---------------------------------------------------------------------------------|
+| Arduino App Lab            | App development                                       | Not public at this time.                                                        |
+| Arduino App CLI            | App management                                        | [arduino/arduino-app-cli](https://github.com/arduino/arduino-app-cli)           |
+| Arduino CLI                | Compile and upload sketch, manage sketch dependencies | [arduino/arduino-cli](https://github.com/arduino/arduino-cli)                   |
+| arduino-router             | Linux communication w/ sketch                         | [arduino/arduino-router](https://github.com/arduino/arduino-router)             |
+| remoteocd                  | Upload tool (hosted mode)                             | [arduino/remoteocd](https://github.com/arduino/remoteocd)                       |
+| App examples               | Documentation                                         | [arduino/app-examples](https://github.com/arduino/app-examples)                 |
+| Preinstalled Bricks        | App components                                        | [arduino/preinstalled-bricks](https://github.com/arduino/preinstalled-bricks)   |
+| arduino/ArduinoCore-zephyr | "UNO Q Board" platform                                | [arduino/ArduinoCore-zephyr](https://github.com/arduino/ArduinoCore-zephyr)     |
```

Comment:
```suggestion
| arduino/ArduinoCore-zephyr | "Arduino UNO Q Board" platform                        | [arduino/ArduinoCore-zephyr](https://github.com/arduino/ArduinoCore-zephyr)     |
```

I accidentally left off the "Arduino" when I wrote that part, but the official name is "Arduino UNO Q Board":

<img width="324" height="269" alt="Image" src="https://github.com/user-attachments/assets/f18f6fbc-076b-4c3a-9ac0-cd447f3812f1" />